fcomm/src/lib.rs

```diff
@@ -1183,7 +1185,11 @@ mod test {
         std::env::set_var(fcomm_path_key, fcomm_path_val.clone());
         assert_eq!(
             std::env::var(fcomm_path_key),
-            Ok(fcomm_path_val.into_os_string().into_string().unwrap())
+            Ok(fcomm_path_val
+                .clone()
+                .into_os_string()
+                .into_string()
+                .unwrap())
```

Comment:
This is the typical kind of annoyance [camino](https://docs.rs/camino/latest/camino/) would eliminate.